source/Government.cpp

```diff
@@ -323,6 +341,10 @@ void Government::Load(const DataNode &node)
 			child.PrintTrace("Error: Expected key to have a value:");
 		else if(key == "player reputation")
 			initialPlayerReputation = add ? initialPlayerReputation + child.Value(valueIndex) : child.Value(valueIndex);
+		else if(key == "reputation max")
+			reputationMax = add ? reputationMax + child.Value(valueIndex) : child.Value(valueIndex);
+		else if(key == "reputation min")
+			reputationMin = add ? reputationMin + child.Value(valueIndex) : child.Value(valueIndex);
```

Comment:
Since we're adding these new attributes, I don't think we should support both child and grandchild for these attributes. It makes sense to have player reputation in both locations since it's already an existing attribute, but the same doesn't hold for new stuff.